WultraMobileTokenSDK/Operations/Service/WMTOperationsImpl.swift

```diff
@@ -82,6 +82,8 @@ public extension WMTErrorReason {
     static let operations_authExpired = WMTErrorReason(rawValue: "operations_authExpired")
     /// Operation has expired when trying to reject the operation.
     static let operations_rejectExpired = WMTErrorReason(rawValue: "operations_rejectExpired")
+    /// Operation authentication failed  (e.g. incorrect totp).
```

Comment:
I think that this can happen on reject as well if I remember correctly from the BE PR, can you confirm?

Reply:
true:
https://github.com/wultra/enrollment-server/pull/901/commits/c8a7250aed9f8b7b9d9f9931860cbcb001ae2a7b

Reply:
So then the comment should not be authentication but more generic like "action"